sqlglot/dialects/bigquery.py

```diff
@@ -215,6 +215,12 @@ def _unix_to_time_sql(self: BigQuery.Generator, expression: exp.UnixToTime) -> s
     return ""
 
 
+def _parse_time(args: t.List) -> exp.Func:
+    if len(args) == 1:
+        return exp.TsOrDsToTime(this=args[0])
+    return exp.Anonymous(this="TIME", expressions=args)
```

Comment:
why is this anonymous? why don't we have a first class expression for this?

Reply:
Decided to use this as a fallback so that the scope of the PR is smaller and the patch is more conservative. It's not straightforward to transpile all variants so I thought we can start with the single argument one and iterate from there if ever needed..

Reply:
ok

Reply:
For example we can do the 3 argument variant in a followup PR, since it'll enable us to transpile it to T-SQL's [TIMEFROMPARTS](https://learn.microsoft.com/en-us/sql/t-sql/functions/timefromparts-transact-sql?view=sql-server-ver16#syntax) call, but I'd probably create a different expression for that like `TimeFromParts` since we already have `DateFromParts`

Reply:
ok